ibis/backends/duckdb/tests/test_catalog.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pandas as pd
+import pandas.testing as tm
+import pytest
+
+import ibis
+import ibis.common.exceptions as exc
+
+
+@pytest.fixture(scope="session")
+def external_duckdb_file(tmpdir_factory):  # pragma: no cover
+    ddb_path = str(tmpdir_factory.mktemp("data") / "starwars.ddb")
+    con = ibis.duckdb.connect(ddb_path)
+
+    starwars_df = pd.DataFrame(
+        {
+            "name": ["Luke Skywalker", "C-3PO", "R2-D2"],
+            "height": [172, 167, 96],
+            "mass": [77.0, 75.0, 32.0],
+        }
+    )
+    con.create_table("starwars", obj=starwars_df)
+    con.disconnect()
+
+    yield ddb_path, starwars_df
+
+    Path(ddb_path).unlink()
```

Comment:
I would just let it live. It's in a temp dir, that seems good enough for now.

Reply:
alternatively you can try to delete and ignore, but i think leaving the file around is fine too